pkg/compute/store/boltdb/utils.go

```diff
@@ -6,14 +6,19 @@ import (
 	bolt "go.etcd.io/bbolt"
 )
 
+const (
+	// uint64ByteSize is the number of bytes needed to represent a uint64
+	uint64ByteSize = 8
+)
+
 // strToBytes converts a string to a byte slice
 func strToBytes(s string) []byte {
 	return []byte(s)
 }
 
 // uint64ToBytes converts an uint64 to a byte slice
 func uint64ToBytes(i uint64) []byte {
-	buf := make([]byte, 8) //nolint:gomnd
+	buf := make([]byte, uint64ByteSize) //nolint:gomnd
```

Comment:
I think `gomnd` was renamed to `mnd` in later versions and that is why //nolint is not working here

Reply:
Replaced gomnd with mnd where appropriate